packages/components/src/components/icon/icons.js

```diff
@@ -1082,3 +1082,5 @@ import './wallet/ic-wallet-upgrade-completed.svg';
 import './wallet/ic-wallet-upgrade-logout.svg';
 import './wallet/ic-wallet-usd-coin-dark.svg';
 import './wallet/ic-wallet-usd-coin-light.svg';
+import './wallet/ic-wallet-xrp-dark.svg';
+import './wallet/ic-wallet-xrp-light.svg';
```

Comment:
There is XRP icon in quill icons:
`PaymentMethodXrpBrandIcon`
`PaymentMethodXrpWhiteIcon `

Reply:
yes @aizad-deriv 
I'm using XRP currency icons from quill in `packages/wallets/src/constants/icons.ts`. 
The other place where we are using icons `as name` is in `useStoreWalletAccountsList`. That's why i added svg icons. In order to replace them with quill icons we need to refactor the hook. For now i didn't do that.